src/Bridge/Symfony/Validator/Serializer/ConstraintViolationListNormalizer.php

```diff
@@ -22,7 +22,7 @@
  */
 final class ConstraintViolationListNormalizer implements NormalizerInterface
 {
-    const FORMAT = 'hydra-error';
+    const FORMAT = ['swagger-error', 'hydra-error'];
```

Comment:
IMO Hydra and Swagger support should be 100% independent and the resulting documents shouldn't mix both formats.